VRCFaceTracking/OSC/OSCMain.cs

```diff
@@ -70,8 +70,36 @@ private void Recv()
                 return;
             }
             var newMsg = new OscMessage(buffer);
-            if (newMsg.Address == "/avatar/change")
-                ConfigParser.ParseNewAvatar((string) newMsg.Value);
+            switch (newMsg.Address)
+            {
+                case "/avatar/change":
+                    ConfigParser.ParseNewAvatar((string)newMsg.Value);
+                    break;
+                case "/avatar/parameters/EyeTrackingActive":
+                    if (UnifiedLibManager.EyeStatus != ModuleState.Uninitialized)
+                    {
+                        if (!(bool)newMsg.Value)
+                            UnifiedLibManager.EyeStatus = ModuleState.Idle;
+                        else UnifiedLibManager.EyeStatus = ModuleState.Active;
+                    }
+                    break;
+                case "/avatar/parameters/LipTrackingActive":
+                    if (UnifiedLibManager.ExpressionStatus != ModuleState.Uninitialized)
+                    {
+                        if (!(bool)newMsg.Value)
+                            UnifiedLibManager.ExpressionStatus = ModuleState.Idle;
+                        else UnifiedLibManager.ExpressionStatus = ModuleState.Active;
+                    }
+                    break;
```

Comment:
If `LipTrackingActive` and `ExpressionTrackingActive` are to be handled identically, would it make sense to remove this and let the case fall through into the next?

Reply:
They are both meant to change the state of the Lip/Expression module so they could both fall into the same case.

Reply:
Changed in https://github.com/benaclejames/VRCFaceTracking/pull/109/commits/e8bc944abd2a9c15d890c1ad6cae5e14462167c5